product_configurator/models/product_attribute.py

```diff
@@ -100,6 +101,19 @@ def check_searchable_field(self):
                   self.custom_type)
             )
 
+    def validate_custom_val(self, val):
+        """ Pass in a desired custom value and ensure it is valid.
+        Probaly should check type, etc, but let's assume fine for the moment.
+        """ 
+        self.ensure_one()
+        if self.custom_type in ('int', 'float'):
+            if self.min_val or self.max_val:
+                val = literal_eval(val)
+                if val < self.min_val or val > self.max_val:
```

Comment:
In these situations, I feel there are rarely "open ended" scenarios.  In theory, there should be validation that max >= min at time of entry.  If both are 0, then this implies true freeform.  There may be scenarios where a negative value is required, meaning min might be -10, and max 0.